pkg/models/error.go

```diff
@@ -17,3 +17,12 @@ type NamespaceNotFoundError struct {
 func (e *NamespaceNotFoundError) Error() string {
 	return fmt.Sprintf("namespace not found: %s", e.Namespace)
 }
+
+// TODO: this should be picked up in a general server error handler
+type GenericUserError struct {
```

Comment:
when should we use this?

Reply:
I just added it for cases where I didn't want to type out a domain specific error